frontmatter descriptions for tracing section (#31706)
DataDog/documentation: content/en/tracing/guide/span_and_trace_id_format.md

---
title: Trace and Span ID Formats

> description: Understand the valid formats for trace and span IDs supported by Datadog tracing libraries and how they correlate with logs.

Reviewer: ```suggestion
description: Guide on valid trace and span ID formats supported by Datadog tracing libraries and how they correlate with logs.
```
